api/src/main/java/org/pmiops/workbench/api/AppController.java

```diff
@@ -33,4 +79,10 @@ public ResponseEntity<EmptyResponse> updateApp(String workspaceNamespace, App ap
   public ResponseEntity<ListAppsResponse> listApp() {
     throw new UnsupportedOperationException("API not supported.");
   }
+
+  private DbWorkspace lookupWorkspace(String workspaceNamespace) throws NotFoundException {
```

Comment:
possibly move this method to workspaceservice, and rename to lookupWorkspaceByNamespace